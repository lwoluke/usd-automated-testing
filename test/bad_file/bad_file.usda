#usda 1.0

def Xform "Root"
{
    def Mesh "BadMesh"
    {
        float3[] points = [(1,1,1), (2,2,2)]
    }
    def Shader "BadShader"
    {
    }
    def "VariantPrim" {
        variants = {}
    }
}
