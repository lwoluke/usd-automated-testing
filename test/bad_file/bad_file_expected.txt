Runtime Error: in _RaiseErrorPEGTL at line 48 of /builds/omniverse/usd-ci/USD/pxr/usd/sdf/textParserHelpers.h -- /mnt/c/Users/lwolu/OneDrive/Documents/Coding/dev/usd-automated-testing/test/bad_file/bad_file.usda:13:9: Expected } at 'variants = {}' in </Root/VariantPrim>

Runtime Error: in Open at line 1052 of /builds/omniverse/usd-ci/USD/pxr/usd/usd/stage.cpp -- Failed to open layer @/mnt/c/Users/lwolu/OneDrive/Documents/Coding/dev/usd-automated-testing/test/bad_file/bad_file.usda@
Failed to open USD file. Ensure the file path is correct and the file is accessible.
